scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -281,7 +281,7 @@ const GROUPS = [
             {
                 label: "Shape Type",
                 type: "dropdown",
-                options: { "box": "Box", "sphere": "Sphere", "ellipsoid": "Ellipsoid", 
```

Comment:
this removes the "ellipsoid" option from Create because you can do the same exact thing with a non-uniformly scaled sphere, so we can simplify the options.  you could still set the shapeType to "ellipsoid" via another script and it would work the same way.